dsf/org.eclipse.cdt.dsf.ui/src/org/eclipse/cdt/dsf/debug/ui/actions/DsfSteppingModeTarget.java

```diff
@@ -26,23 +26,21 @@
  *
  * @since 1.0
  */
-public class DsfSteppingModeTarget implements ISteppingModeTarget, ITargetProperties {
+public class DsfSteppingModeTarget implements ISteppingModeTarget {
```

Comment:
Please check, I guess org.eclipse.cdt.dsf.ui/META-INF/MANIFEST.MF version should be increased too. Last change was 6 months ago probably it is already released so bundle version must be bumped.

Reply:
Fixed.